ui/components/app/signature-request-original/signature-request-original.component.js

```diff
@@ -177,8 +177,12 @@ export default class SignatureRequestOriginal extends Component {
                 className="request-signature__row"
                 key={`request-signature-row-${index}`}
               >
-                <div className="request-signature__row-title">{`${name}:`}</div>
-                <div className="request-signature__row-value">{value}</div>
+                <div className="request-signature__row-title">
+                  {sanitizeString(`${name}:`)}
```

Comment:
Are any of these component changes related to the task? If not, can you extract them into another PR?

Reply:
They are the rebase legacies, removed 🙏